compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/TextFieldKeyInput.kt

```diff
@@ -130,7 +143,7 @@ internal class TextFieldKeyInput(
                             0
                         )
                     }?.apply()
-                KeyCommand.DELETE_NEXT_CHAR -> {
+                KeyCommand.DELETE_NEXT_CHAR ->
```

Comment:
https://github.com/JetBrains/compose-multiplatform-core/blob/26c91eff74c367dd5959a9a174f258f0e8dcbeeb/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/TextFieldKeyInput.kt#LL159C16-L194C22

Reply:
> Could you please keep curly brackets in a such multi-line block?

It would be a good change if we change our code, but this is commonMain, and better to not change it if not needed (to avoid merge conflicts)